src/index.html

```diff
@@ -1,17 +1,182 @@
 <!doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+    <meta name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="POST"
+    onsubmit="onSubmit"
+    >
+      <fieldset>
+        <legend>Personal information</legend>
+        <label for="surname">
+            Surname:
+        </label>
+        <input type="text"
+          name="Surname"
+          id="surname"
+          >
+        <br>
+
+        <label for="name">
+            Name:
+        </label>
+        <input type="text"
+            name="name"
+            id="name"
+          >
```

Comment:
```suggestion
        <input
          type="text"
          name="name"
          id="name"
        >
```
Закриваюча дужка має бути на тому ж рівні відступів, що і відкриваюча.
Кожен атрибут на новому рядку